.prettierrc

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
What is this for?

Reply:
So editors integrated with prettier know what are the conventions on this repo. We use single quote as convention everywhere else so in my case prettier was changing the whole file on save.